go/packages/dev-mode/agent/forwarder.go

```diff
@@ -143,6 +147,36 @@ func FindReqData(logs []LogItem) *LogItem {
 	return nil
 }
 
+func IsDefaultConsoleMessage(record interface{}) bool {
+	message := fmt.Sprintf("%v", record)
+	arr := strings.Split(message, "\t")
+	// Not enough items in the list
+	// The first item is not a date timestamp
+	// The second item is not a request id
+	// The third item is not WARN or ERROR
+	if !hasInternalExtension || len(arr) < 4 || !IsValidDate(arr[0]) || !IsValidUUID(arr[1]) || !IsWarnOrError(arr[2]) {
+		return false
+	}
+	return true
+}
+
+func IsWarnOrError(logLevel string) bool {
+	if logLevel == "WARN" || logLevel == "ERROR" {
+		return true
```

Comment:
Will it filter all `console.error` calls? If so, it can be problematic, as we issue captured events only for those which host Error instance.  So if users log strings with `console.error` they won't appear in dev mode

Reply:
Ahh yes this is a good point 🤔 

I will do a bit more research for errors

Reply:
@medikoo This is actually kind of a problem then. There doesn't seem to be a straight forward way to pull out the error from `console.error` from the message alone. Especially if you pass multiple messages into the error function 👇 

```javascript
console.error('string 1', new Error('This is a bug'), 'string 2');
```

I could log the messages up to when the error object was printed but it would be difficult, maybe impossible to log the messages behind the error 🤔 I'll post this question in our linear ticket.